build/dockerfiles/rhel.Dockerfile

```diff
@@ -20,10 +20,13 @@ USER 0
 ################# 
 
 ARG BOOTSTRAP=false
+ENV BOOTSTRAP=${BOOTSTRAP}
 ARG LATEST_ONLY=false
+ENV LATEST_ONLY=${LATEST_ONLY}
```

Comment:
You shouldn't need env vars here, the ARGs are available during the docker build.